server/constants.py

```diff
@@ -5,6 +5,13 @@
 STAFF_ROLE = 'staff'
 INSTRUCTOR_ROLE = 'instructor'
 LAB_ASSISTANT_ROLE = 'lab assistant'
+ROLE_DISPLAY_NAMES = {
+    'student' : 'Student',
+    'grader' : 'Reader',
+    STAFF_ROLE : 'Teaching Assistant',
```

Comment:
There should not be a space before the colon. Also, what about the rest of the repeated string literals?